web/libs/app-common/src/pages/AccountSettings/sections/Hotkeys.jsx

```diff
@@ -0,0 +1,599 @@
+import { useEffect, useState } from "react";
```

Comment:
```suggestion
import { useEffect, useState, useMemo, useReact } from "react";
```

this file misses memoization pattern, which results in the degraded performance and uneccessary rerenders. I'd suggest wrapping places with `useMemo` and `useCallback` where it is necessary for:
- prevent recomputations of potentially expensive operations (e.g. `getModifiedHotkeys`)
- prevent components rerenders due to non-memoized handlers (e.g. handlers in `<HotkeySection>`)